core/src/main/java/greencity/controller/AdviceController.java

```diff
@@ -79,6 +78,23 @@ public LanguageTranslationDTO getRandomAdviceByHabitIdAndLanguage(
         return adviceService.getAllAdvices();
     }
 
+    /**
+     * The controller which returns advice by id {@link Advice}.
+     *
+     * @return instance of {@link AdviceVO}
+     * @author Markiyan Derevetskyi
+     * */
+    @ApiOperation("Get advice by id")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = HttpStatuses.OK),
+            @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+    })
+    @GetMapping("/{adviceId}")
```

Comment:
/{adviceId} -> /{id}. It is better by conventions. You already know this is advice controller, so no need to add advice prefix

Reply:
Changed